app/views/email_templates/index.html.slim

```diff
@@ -0,0 +1,22 @@
+h1 Alle E-Mailvorlagen
+
+= form_navigation_btn :new
+
+.table-responsive
+  table.table
+    thead
+      th Betreff
+      th Text
+      th Art
+      th Aktiv
+      th
+    tbody
+      - @email_templates.each do |email_template|
```

Comment:
some suggestions for the index view & table:
- order of the cells: `Art | Aktiv | Betreff | Text`
- possibility to filter `Art` ?
- you can use `boolean_glyph(email_template.active)` to save some space
- is there a possibility to set the with for the subject or body?
    if you have a big text the subject gets squashed and is userunfriendly to read:
<img width="994" alt="screen shot 2017-11-24 at 12 00 11" src="https://user-images.githubusercontent.com/11731325/33207731-1b64b83a-d10f-11e7-82e9-5ed788bf7d00.png">

Reply:
There was no request for filtering. 

Regarding the big text possibility, these are just templates, so I doubt there will contain big texts. Big texts are not easily reusable (reusability is the point of having templates).

The icon instead of Ja/Nein, wouldn't save that much of space.

Reply:
The platform should be consistent. The icons where already used for the `client_notifications`.
Regarding the filter, right it was not requested, but makes it once more consistent and more user friendly - up to you.